src/modules/launcher/Wox.Core/Resource/FontHelper.cs

```diff
@@ -43,7 +46,9 @@ public static FontStyle GetFontStyleFromInvariantStringOrNormal(string value)
             {
                 return (FontStyle)_fontStyleConverter.ConvertFromInvariantString(value);
             }
+#pragma warning disable CA1031 // Do not catch general exception types
```

Comment:
A similar comment to part #1 of this PR: It would be better to use attributes to suppress this error. You can find an example here: https://github.com/microsoft/PowerToys/pull/6958/files#diff-82046de380447e40c4e6c040bdb9643d4f4c019d934ef9c310eba7920891b27aR333